cmake/impl_opengl3.cmake

```diff
@@ -0,0 +1,44 @@
+if(IMGUI_IMPL_OPENGL3)
+    set(IMGUI_IMPL_OPENGL_ES "none" CACHE STRING "OpenGL ES version")
+    set_property(CACHE IMGUI_IMPL_OPENGL_ES PROPERTY STRINGS none 2 3)
+
+    set(IMGUI_IMPL_OPENGL_LOADER "GLEW" CACHE STRING "OpenGL Loader")
+    set_property(CACHE IMGUI_IMPL_OPENGL_LOADER PROPERTY STRINGS GLEW GL3W GLAD Custom)
+    option(IMGUI_IMPL_OPENGL_LOADER_CUSTOM "OpenGL custom loader" "")
+
+    message(STATUS "IMGUI_IMPL_OPENGL_ES: ${IMGUI_IMPL_OPENGL_ES}")
+    if(IMGUI_IMPL_OPENGL_ES STREQUAL 2)
+        add_definitions(-DIMGUI_IMPL_OPENGL_ES2)
+    elseif(IMGUI_IMPL_OPENGL_ES STREQUAL 3)
+        add_definitions(-DIMGUI_IMPL_OPENGL_ES3)
```

Comment:
```cmake
else()
  message(FATAL_ERROR "...")
endif()
```
?

Reply:
it can be none (or anything else), so fatal not required